source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -1742,6 +1755,7 @@ def event_alert(self):
 	(None,oleacc.ROLE_SYSTEM_PROPERTYPAGE):"Dialog",
 	(None,oleacc.ROLE_SYSTEM_GROUPING):"Groupbox",
 	("TrayClockWClass",oleacc.ROLE_SYSTEM_CLIENT):"TrayClockWClass",
+	("TrayClockWClass",oleacc.ROLE_SYSTEM_CLOCK):"TrayClockWClass",

```

Comment:
I recall testing this on 7, but not the others. I also don’t have easy access to a Windows 7 machine anymore, unfortunately.

 

From: josephsl [mailto:notifications@github.com] 
Sent: Sunday, April 3, 2016 10:13
To: nvaccess/nvda <nvda@noreply.github.com>
Cc: Davy Kager <mail@davykager.nl>
Subject: Re: [nvaccess/nvda] Improve reading of the clock in the taskbar (#5751)

 

In source/NVDAObjects/IAccessible/__init__.py <https://github.com/nvaccess/nvda/pull/5751#discussion_r58306154> :

> @@ -1742,6 +1755,7 @@ def event_alert(self):
>       (None,oleacc.ROLE_SYSTEM_PROPERTYPAGE):"Dialog",
>       (None,oleacc.ROLE_SYSTEM_GROUPING):"Groupbox",
>   ("TrayClockWClass",oleacc.ROLE_SYSTEM_CLIENT):"TrayClockWClass",
> + ("TrayClockWClass",oleacc.ROLE_SYSTEM_CLOCK):"TrayClockWClass",

I hope this doesn't break in Vista, 7 and 8.x.

—
You are receiving this because you authored the thread.
Reply to this email directly or  <https://github.com/nvaccess/nvda/pull/5751/files/19b600d36da6a6f2557663dc605a251fad2086c6#r58306154> view it on GitHub  <https://github.com/notifications/beacon/AC9y8c9oG1KiRxHgXl3PcGB8zOsJQTF5ks5pz3amgaJpZM4HZsGx.gif>